### ISSUE TITLE
Upgrading container tool with Drawer and JD abilities

### ISSUE THREAD

**Issue by Klaranth** [pandorabox-io/in-game#186]
UncleSam ; SX ; Huhhila 

A tool to copy-paste drawer's upgrades

Adding to metatool modpack, task for container tool.
Preliminary plans are made to allow similar for technic machine upgrades using container tool.

Looks like I did not have issue for machine upgrades either, created for both. Now that there's 2 things with similar functionality I might try to actually implement both:
https://github.com/S-S-X/metatool/issues/114
https://github.com/S-S-X/metatool/issues/115

Adding jumpdrive. 
Separate from basic technic machines so should add another issue for that. 
Bit more complicated with different types of storage.

Basically I built metatool stuff from beginning with idea that it should allow easily supporting any nodes without having to care at all about differences between nodes, that's why for example travelnets, geocaches, lcd screens, lua controllers and books are supported by single tool.
(by oversimplified definition it is basically interface translation tool)
These are very good upgrades for container tool as currently I think it is probably least useful tool in set.

**Comment by S-S-X** [comment]
Issue for machines: https://github.com/S-S-X/metatool/issues/114
Issue for drawers: https://github.com/S-S-X/metatool/issues/115
Issue for jumpdrive: https://github.com/S-S-X/metatool/issues/116

Pull request (drawers, machines and jumprdrive implemented): https://github.com/S-S-X/metatool/pull/117

Needs testing (and possibly fixing problems if any), if you happen to have extra time feel free to fetch branch containertool-upgrade-handling to try it and report any issues to https://github.com/S-S-X/metatool/pull/117

**Comment by S-S-X** [comment]
https://github.com/pandorabox-io/pandorabox-mods/pull/1999

**Comment by Klaranth** [comment]
Huhhila ; SX
Would areatool for metatool be too foreign concept, if it were to set /area_pos1 and /area_pos2 ? And read/calculate values for those by clicking JDs, various protboxes, quarries, ...?

Managing and creating areas would be easily possible with metatool easily yes but tool API is specialized more for metadata translation and manipulation than generic stuff. So could be easily done but not sure how good idea it'll be (see sharetool, it does similar things besides basic metadata translation / display)
